docs/user-guide/testing-models.rst

```diff
@@ -2,9 +2,9 @@
        library Rasa Core to improve the dialogue management of your contextual
        AI Assistant.
 
-.. _evaluating-models:
+.. _testing-models:
 
-Evaluating Models
+Testing Models
```

Comment:
would also leave as "evaluating models"

Reply:
I guess that was one of the main ideas behind this change.

Reply:
but surely not *everything*? calling them `end-to-end tests` totally makes sense, but i don't think that means we need to say "evaluating" literally everywhere